[2025-06-12T01:40:17.486+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dynamic_training_pipeline.merge_data manual__2025-06-12T01:40:12.747699+00:00 [queued]>
[2025-06-12T01:40:17.489+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dynamic_training_pipeline.merge_data manual__2025-06-12T01:40:12.747699+00:00 [queued]>
[2025-06-12T01:40:17.489+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-12T01:40:17.493+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): merge_data> on 2025-06-12 01:40:12.747699+00:00
[2025-06-12T01:40:17.497+0000] {standard_task_runner.py:60} INFO - Started process 264 to run task
[2025-06-12T01:40:17.498+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dynamic_training_pipeline', 'merge_data', 'manual__2025-06-12T01:40:12.747699+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/dag_dynamic.py', '--cfg-path', '/tmp/tmpv4q6zc5v']
[2025-06-12T01:40:17.500+0000] {standard_task_runner.py:88} INFO - Job 113: Subtask merge_data
[2025-06-12T01:40:17.518+0000] {task_command.py:423} INFO - Running <TaskInstance: dynamic_training_pipeline.merge_data manual__2025-06-12T01:40:12.747699+00:00 [running]> on host 88831a01c40c
[2025-06-12T01:40:17.539+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dynamic_training_pipeline' AIRFLOW_CTX_TASK_ID='merge_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-12T01:40:12.747699+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-12T01:40:12.747699+00:00'
[2025-06-12T01:40:17.540+0000] {logging_mixin.py:188} INFO - Archivo no encontrado en raíz: /opt/***/data_1_report.html
[2025-06-12T01:40:17.540+0000] {logging_mixin.py:188} INFO - Archivo no encontrado en raíz: /opt/***/data_2_report.html
[2025-06-12T01:40:17.540+0000] {logging_mixin.py:188} INFO - No hay datos para concatenar.
[2025-06-12T01:40:17.540+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-12T01:40:17.543+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dynamic_training_pipeline, task_id=merge_data, execution_date=20250612T014012, start_date=20250612T014017, end_date=20250612T014017
[2025-06-12T01:40:17.553+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-12T01:40:17.563+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
